tests/conftest.py

```diff
@@ -74,6 +74,15 @@ def one_goal(app):
     db.session.add(new_goal)
     db.session.commit()
 
+#This fixture gets called in tests for returning goals sorted by title
+@pytest.fixture
+def three_goals(app):
```

Comment:
You should probably also have a fixture for 3 tasks.  

It's great however that you added your own fixture.